storage/uri.go

```diff
@@ -22,7 +26,23 @@ func (u *uri) Extension() string {
 }
 
 func (u *uri) MimeType() string {
-	return "x/unknown" // TODO - added in #405
+	ext := filepath.Ext(u.raw[1:])
```

Comment:
Why is the first character being trimmed from the URI?

Reply:
More to the point, why not use u.Extension() ;)

Reply:
the first character isn't being trimmed from the URI.  The first character, which is "." is being trimmed from the extension.  We can use uri.Extension.

Reply:
Are you sure? It looks very much like the raw uri string is being sliced then passed to the extension function of filepath

Reply:
on second thought I see this didn't translate well in the move and now is indeed just removing the first character of the URI. Will fix.